x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/find.ts

```diff
@@ -22,475 +22,952 @@ export default function createFindTests({ getService }: FtrProviderContext) {
 
     afterEach(() => objectRemover.removeAll());
 
-    for (const scenario of UserAtSpaceScenarios) {
```

Comment:
Hmm. I wonder, if instead of duplicating tests here, we add a wrapper for each tests that passes down configuration, and each test is updated to use the configuration.

I did this for ephemeral support in the alerting task runner.

See [the wrapper function](https://github.com/elastic/kibana/blob/main/x-pack/plugins/alerting/server/task_runner/task_runner.test.ts#L123) and [how it's used](https://github.com/elastic/kibana/blob/main/x-pack/plugins/alerting/server/task_runner/task_runner.test.ts#L408).

It feels better in that someone doesn't need to update multiple, duplicate tests for a change that affects both scenarios.

WDYT?

Reply:
make sense!